src/Goerli-DssSpell.t.base.sol

```diff
@@ -139,8 +132,12 @@ interface ArbitrumTeleportBridgeLike is TeleportBridgeLike {
     function inbox() external view returns (address);
 }
 
-contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
-    Hevm hevm;
+contract GoerliDssSpellTestBase is Config, Test, DSMath {
+
+    string config;
+    Domain rootDomain;
+    Domain optimismDomain;
+    Domain arbitrumDomain;
```

Comment:
We probably want to use the `ArbitrumDomain` and `OptimismDomain` types here so that we can replay txs from L1 to L2. IE the `XXXDomain.sendToHost(...)` call for full integration testing. This will help us to verify that the L2 part of the spell has been queued up properly in the CTC.

Reply:
My thought here was that since we didn't have a specific use for this right now in the repo, leave it basic and then add these when we have the need.  That being said, if you feel strongly i can put the code in since it would mostly be copy/paste from your work

Reply:
Well the whole point of this addition is we can do integration tests for Optimism and Arbitrum. :)

Also, as I said in the primary comment I think you can just import them from `dss-test` instead of copying in. If you need code changes (such as selecting the block) it should be merged upstream to `dss-test` if it's missing functionality.

Reply:
I'd also think we should be including `dss-test`. This repo should be just another one that takes usage of that one.
We are still on development phases of `dss-test` so whatever is needed for this repo we should be able to make sure is compatible.

Reply:
so the problem with `dss-test` is that it is using `>= 0.8.0` which we have so far wanted to keep `spells-*` and `dss-exec-lib` at `0.6.12` for safety reasons.  If we want to use `dss-test` then we need to upgrade those and that should be a broader team discussion/decision

Reply:
Ahh yes good point. Maybe it's time to look at upgrading the spells repo version, but that's definitely a bigger change.

Reply:
I don't really think there is a safety reason to not upgrade the spells to 0.8.x, more if we are doing it without optimizations.
Regarding `DssExecLib`, remember we can keep the deployed library with solc 0.6.12 and just use the new solidity version when inheriting `DssExec` and `DssAction`.

Reply:
> Regarding `DssExecLib`, remember we can keep the deployed library with solc 0.6.12 and just use the new solidity version when inheriting `DssExec` and `DssAction`.

You're saying we don't have to update the deployed contract, but we do need to still update the repo, correct?